tox.ini

```diff
@@ -1,6 +1,21 @@
 [tox]
-envlist = py37
+envlist = py37, py38, py39, py310, py311
+isolated_build = true
+
+[gh-actions]
+python =
+    3.8: py38, 
+    3.9: py39, flake8
+    3.10: py310,
+    3.11: py311,
```

Comment:
How GitHub actions are related to tox ?

Reply:
tox-gh-actions is a tox plugin which helps running tox on GitHub Actions with multiple different Python versions on multiple workers in parallel. This project is inspired by [tox-travis](https://github.com/tox-dev/tox-travis).

https://pypi.org/project/tox-gh-actions/